os_stub/armbuild_lib/div64.c

```diff
@@ -0,0 +1,166 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2021-2022 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+/*the code URL: https://github.com/llvm/llvm-project/blob/main/compiler-rt/lib/builtins/udivmodti4.c */
+
+typedef int si_int;
+typedef unsigned su_int;
+
+typedef          long long di_int;
+typedef unsigned long long du_int;
+
+typedef int ti_int;
+typedef unsigned tu_int;
+
+#ifndef CHAR_BIT
+#define CHAR_BIT 8
+#endif
+
+typedef union
+{
+    tu_int all;
+    struct
+    {
+#if _YUGA_LITTLE_ENDIAN

```

Comment:
What is YUGA?

Reply:
The `_YUGA_LITTLE_ENDIAN` is from: https://github.com/llvm/llvm-project/blob/542977d43841820614a32823c33415042430e230/compiler-rt/lib/builtins/int_endianness.h. I think it may be a custom Macro usage.